src/library/Uno.Themes.WinUI.Markup/Theme/CheckBox.cs

```diff
@@ -0,0 +1,254 @@
+							public static ResourceValue<Brush> IndeterminatePressed => new("CheckBoxGlyphForegroundIndeterminatePressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminateDisabled")]
+							public static ResourceValue<Brush> IndeterminateDisabled => new("CheckBoxGlyphForegroundIndeterminateDisabled", true);
+						}
+
+						public static class PathData
+						{
+							[ResourceKeyDefinition(typeof(string), "CheckBoxHyphenGlyphPathData")]
+							public static ResourceValue<string> Hyphen => new("CheckBoxHyphenGlyphPathData", true);
+
+							[ResourceKeyDefinition(typeof(string), "CheckBoxCheckGlyphPathData")]
+							public static ResourceValue<string> Check => new("CheckBoxCheckGlyphPathData", true);
+						}
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "CheckBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("CheckBoxFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(FontWeights), "CheckBoxFontWeight")]
+						public static ResourceValue<FontWeights> FontWeight => new("CheckBoxFontWeight", true);
+
+						[ResourceKeyDefinition(typeof(double), "CheckBoxFontSize")]
+						public static ResourceValue<double> FontSize => new("CheckBoxFontSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "CheckBoxCharacterSpacing")]
+						public static ResourceValue<double> CharacterSpacing => new("CheckBoxCharacterSpacing", true);
+					}
+
+					public static class FocusArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxFocusAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxFocusAreaSize", true);
+						}
+					}
+
+					public static class CheckArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxCheckAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxCheckAreaSize", true);
+						}
+
+						public static class Length
+						{
+							[ResourceKeyDefinition(typeof(GridLength), "CheckBoxCheckAreaLength")]
+							public static ResourceValue<GridLength> Default => new("CheckBoxCheckAreaLength", true);
+						}
+
+						public static class CornerRadius
+						{
+							[ResourceKeyDefinition(typeof(MUXCornerRadius), "CheckBoxCheckAreaCornerRadius")]
+							public static ResourceValue<MUXCornerRadius> Default => new("CheckBoxCheckAreaCornerRadius", true);
+						}
+
+						public static class BorderThickness
+						{
+							[ResourceKeyDefinition(typeof(Thickness), "CheckBoxCheckAreaBorderThickness")]
+							public static ResourceValue<Thickness> Default => new("CheckBoxCheckAreaBorderThickness", true);
+						}
+
+						public static class Padding
+						{
+							[ResourceKeyDefinition(typeof(Thickness), "CheckBoxCheckAreaPadding")]
+							public static ResourceValue<Thickness> Default => new("CheckBoxCheckAreaPadding", true);
+						}
```

Comment:
These should just be straight properties instead of having a `Default`,

So, CheckArea.Size, CheckArea.Padding, etc.

Reply:
removed wrapper classes and named contained properties after them